cmake/FindSTM32HAL.cmake

```diff
@@ -1,176 +1,190 @@
+    
+        SET(HAL_PREFIX stm32f2xx_)
+    
+    ELSEIF(STM32_FAMILY STREQUAL "F3")
+        SET(HAL_COMPONENTS adc can cec comp cortex crc dac dma flash gpio i2c i2s
+                           irda nand nor opamp pccard pcd pwr rcc rtc sdadc
+                           smartcard smbus spi sram tim tsc uart usart wwdg)
+    
+        SET(HAL_REQUIRED_COMPONENTS cortex pwr rcc)
+    
+        SET(HAL_EX_COMPONENTS adc crc dac flash i2c i2s opamp pcd pwr
+                              rcc rtc smartcard spi tim uart)
+    
+        SET(HAL_PREFIX stm32f3xx_)
+    
+    ELSEIF(STM32_FAMILY STREQUAL "F4")
+        SET(HAL_COMPONENTS adc can cec cortex crc cryp dac dcmi dma dma2d eth flash
+                           flash_ramfunc fmpi2c gpio hash hcd i2c i2s irda iwdg ltdc
+                           nand nor pccard pcd pwr qspi rcc rng rtc sai sd sdram
+                           smartcard spdifrx spi sram tim uart usart wwdg fmc fsmc
+                           sdmmc usb)
+    
+        SET(HAL_REQUIRED_COMPONENTS cortex pwr rcc)
+    
+        # Components that have _ex sources
+        SET(HAL_EX_COMPONENTS adc cryp dac dcmi dma flash fmpi2c hash i2c i2s pcd
+                              pwr rcc rtc sai tim)
+    
+        SET(HAL_PREFIX stm32f4xx_)
+    
+    ELSEIF(STM32_FAMILY STREQUAL "F7")
+        SET(HAL_COMPONENTS adc can cec cortex crc cryp dac dcmi dma dma2d eth flash
+                           gpio hash hcd i2c i2s irda iwdg lptim ltdc nand nor pcd
+                           pwr qspi rcc rng rtc sai sd sdram smartcard spdifrx spi
+                           sram tim uart usart wwdg fmc sdmmc usb)
+    
+        SET(HAL_REQUIRED_COMPONENTS cortex pwr rcc)
+    
+        # Components that have _ex sources
+        SET(HAL_EX_COMPONENTS adc crc cryp dac dcmi dma flash hash i2c pcd
+                              pwr rcc rtc sai tim)
+    
+        SET(HAL_PREFIX stm32f7xx_)
+    
+    ELSEIF(STM32_FAMILY STREQUAL "L0")
+        SET(HAL_COMPONENTS adc comp cortex crc crs cryp dac dma exti firewall flash gpio i2c
+                           i2s irda iwdg lcd lptim lpuart pcd pwr rcc rng rtc smartcard
+                           smbus spi tim tsc uart usart utils wwdg)
+    
+        SET(HAL_REQUIRED_COMPONENTS cortex pwr rcc)
+    
+        # Components that have _ex sources
+        SET(HAL_EX_COMPONENTS adc comp crc cryp dac flash i2c pcd pwr rcc rtc smartcard tim uart usart)
+    
+        SET(HAL_PREFIX stm32l0xx_)
+    ELSEIF(STM32_FAMILY STREQUAL "L1")
+        SET(HAL_COMPONENTS adc comp cortex crc cryp dac dma flash flash_ramfunc
+    					   gpio i2c i2s irda iwdg lcd nor opamp pcd pwr rcc rtc
+    					   sd smartcard spi sram tim uart usart wwdg)
+        SET(HAL_REQUIRED_COMPONENTS cortex pwr)
+        
+        # Components that have _ex sources
+        SET(HAL_EX_COMPONENTS adc cryp dac flash opamp pcd pwr rcc rtc spi tim)
+        # Components that have ll_ in names instead of hal_
```

Comment:
Is the list missing or the comment useless?

Reply:
I don't think it's related to my PR. This comment is present in the current master too.